### ISSUE TITLE
title for test case details page

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-axsdb-web#23]
Page: "details for test h44_0000034 ..."
Issue: make "Screen reader use the for attribute [...]" a sub-title ("Details for Test H44_0000034" should be the prominent heading)

Note: rename "Test" to "Test Case" in the heading too, please
